ordered_model/admin.py

```diff
@@ -174,13 +174,22 @@ def get_queryset(cls, request):
         Returns a QuerySet of all model instances that can be edited by the
         admin site. This is used by changelist_view.
         """
-        qs = cls.model._default_manager.get_query_set()
```

Comment:
The backwards compatibility shims for `get_query_set` were removed in Django 1.8. It should be `get_queryset`.
https://docs.djangoproject.com/en/1.9/releases/1.6/#get-query-set-and-similar-methods-renamed-to-get-queryset